pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -999,7 +1000,7 @@ private void addBrokerAdditionalServlets(WebService webService,
                     ((AdditionalServletWithPulsarService) additionalServlet).setPulsarService(this);
                 }
                 webService.addServlet(servletWithClassLoader.getBasePath(), servletWithClassLoader.getServletHolder(),
-                        config.isAuthenticationEnabled(), attributeMap);
+                        config.isAuthenticationEnabled(), attributeMap, false);
```

Comment:
Can we apply this GzipHandle support for all the endpoints instead of the metric endpoint? It will benefit for all the REST response.